source-reminder/README.md

```diff
@@ -1 +1,76 @@
-# Source Reminder
\ No newline at end of file
+# Source Reminder
+
+> Source Reminder is a [Slack App](https://medium.com/glasswall-engineering/how-to-create-a-slack-bot-using-aws-lambda-in-1-hour-1dbc1b6f021c) that runs as a serverless function at scheduled intervals.
+### Source Diversity Project
+With a commitment to coverage that reflects the diversity of Texas, we have recently put an emphasis on ensuring
+that our sources are representative of the entire state. The idea of the initiative is to track race, ethnicity and gender of the people featured in our work.
+Currently, reporters do so through a Google Form, but the system is not perfect.
+Thus, this tool exists to cross-reference articles on the website and articles submitted via the form to
+identify any pieces of work for which source information has not been submitted.
+
+## Requirements
+* [Lambda](https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc&awsf.Free%20Tier%20Types=tier%23always-free) (Amazon Web Services)
+* [Google Drive](https://www.google.com/drive/)
+* [Slack SDK](https://slack.dev/node-slack-sdk/)
+* [Gspread](https://github.com/burnash/gspread) 
+* [Client library](https://github.com/googleapis/oauth2client) for OAuth 2.0&mdash;also known as `oauth2client`
+* [Python Requests](https://docs.python-requests.org/)
+* [Python Datetime](https://docs.python.org/3/library/datetime.html)
+
+## Google Cloud Integration
+To properly receive the data from an API call, you&rsquo;ll be required at some point to provide what&rsquo;s called a *credentials* file for said account.
+It lives in the Amazon Web Services Lambda computing platform with the rest of the files and responds to webhook requests. Seven libraries are required, but two of them are pre-installed with Python, so really you only need to worry about packaging the five above.
+
+## Deployment
```

Comment:
As discussed on the screenshare, we could simplify the deployment instructions / process by using the Makefile commands (`make deploy`) that you have above.  Here's how we [describe deployment for another tacobot](https://github.com/texastribune/tacobots/tree/master/hoedown-helper#deploy).  For that Makefile deploy command to work, we need to move the currently deployed Lambda function from `us-east-2` to `us-east-1`.  Eventually we'll have a github action and we won't have to worry about this at all in the docs, and I'm also interested in learning more about the SAM CLI tool from AWS - great find.